### ISSUE TITLE
Feature: More chat messages for torches

### ISSUE THREAD

**Issue by MelvinMcSnatch** [Muttley/foundryvtt-shadowdark#201]
There's a message that tells you when a "Torch was lit".

At very least, I think a chat message should pop up when a torch runs out. I would also like to see a countdown of remaining time (custom increments or otherwise).

**Comment by Muttley** [comment]
There *should* be a chat message when a torch expires.  Maybe they're not showing up for some reason.

**Comment by MelvinMcSnatch** [comment]
My light tracker UI macro doesn't do the trick either. I'm not running any other modules or anything. I ran a torch during dinner time to see what would happen and the torch did disappear from the inventory, but the token still had 30 dim, 5 bright. I retested by setting a torch to 1 min and I get the same thing.

**Comment by Muttley** [comment]
> My light tracker UI macro doesn't do the trick either. I'm not running any other modules or anything. I ran a torch during dinner time to see what would happen and the torch did disappear from the inventory, but the token still had 30 dim, 5 bright. I retested by setting a torch to 1 min and I get the same thing. No message either.

Did you have any errors appear in the console?  You can bring that up with F12.

**Comment by MelvinMcSnatch** [comment]
I apologize for not being a good bug reporter. Awesome work btw. Let me try again:

- Using Chome, Firefox, or Edge: I start a torch with 1 minute duration (I had tested it with full 60 minutes as well).
- When the duration ends, there is no message and token light radius remains 30 dim 5 bright.
- The console throws the following error:

> Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'system')
    at ActorSD.toggleLight (ActorSD.mjs:445:9)
    at ActorSD.yourLightWentOut (ActorSD.mjs:365:8)
    at LightSourceTrackerSD._performTick (LightSourceTrackerSD.mjs:303:12)
ActorSD.mjs:376 Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'name')
    at ActorSD.yourLightWentOut (ActorSD.mjs:376:24)
    at LightSourceTrackerSD._performTick (LightSourceTrackerSD.mjs:303:12)

- The lights out macro does toggle light sources off, but does not reset light radius. No console errors from any browser.

**Comment by Muttley** [comment]
Just to confirm, there was a token on a scene in this cases?

> My light tracker UI macro

You mention a macro a couple of times, which macro is this?  Something you've created?  Or are you using the controls on the LIght Tracker interface?

**Comment by MelvinMcSnatch** [comment]
There is a token on the scene. I'm using one of the pregens from the compendium specifically.

I am taking about the Light Tracker controls. I open the Light Tracker dialog with a macro that was included in the compendium. It is not mine.

**Comment by Muttley** [comment]
This was related to #202
